### ISSUE TITLE
[Feature Request: Simple] Open new tab always next to current

### ISSUE THREAD

**Issue by RealDavidoff** [mozilla/multi-account-containers#1385]
This is my review because I love it: https://addons.mozilla.org/en-US/firefox/addon/multi-account-containers/reviews/1325378/

So I thought you forgive me if I make a suggestion for further improvement?

Currently, when clicking a link opens a new tab, the tab 
- either jumps two three positions further right (jumps yes, like it can't decide where it want itself to open)
- or, even worse, the new tab opens at the far right end

As I am one of those naughty web users who have 30+ tabs open (I work multi-tasking, lol),
it is really annoying to have to go all the way to the end of life (tabs) and then move it all the way back to become the current tab's neighbor for easier working on the related tab.

They ARE related because I opened it FROM the current tab, you see.

---
I believe it is ultra easy for the programmer to open tabs RIGHT NEXT to the current, instead of doing one of the two things I pointed out above that are current practice (at least for me/my installation).

Could you pl improve this thingy?

**Comment by stoically** [comment]
Glad you like the Add-on.

Closing this as duplicate of https://github.com/mozilla/multi-account-containers/issues/1030 - please vote there. Thanks!

**Comment by RealDavidoff** [comment]
No, is not duplicate, has nothing to do with that.
#1030 is about reverse order
Mine is about haphazard/random distribution of new tabs.

It appears the full ramifications of opening new tabs haven't been considered for all use cases (version, extensions, whatever).

Reverse order?
That would be a joy. Compared to now.

Again: Logical - and simplest for developers - is: open any new tab RIGHT NEXT to the current.

Those who use the container addon AND choose to open a new tab IN A DIFFERENT container (right click context menu) then won't mind to drag that new tab to where they feel they need it.
This is the minority of users anyway.
Most open a new tab in the same container - if they use container addon at all.

**Comment by stoically** [comment]
Ah, ok, I've misunderstood. Unfortunately I'm unable to reproduce. For me assigned domains always open next to the current tab. Could you provide specific steps to reproduce (which domain, where/how to click links)? Also please make sure that you don't have other Add-ons installed that could be the reason for that to happen.

**Comment by RealDavidoff** [comment]
I agree yes, I mentioned the "discard" addon on purpose, I was already wondering if *together* with "multi account containers" it might be contributing to the issue at hand.

Hmm, "uninstalling other addons", I have to see how to help pinpointing this issue then, because it took a LOT of work to put the addons together and with options that facilitate my work. MAYBE someone lends me a 2nd computer for a clean installation with *only* multi account containers active. Let me check.

Either way I trust you were able to reproduce at least this: that the addon causes new tabs in a *different* container to jump many positions to the right, and not always the same number of positions, making it extremely hard to find the new tab when you have ~60 open. ;-)
(sorry for that)

**Comment by stoically** [comment]
For me happens what is described in #1030 - new tabs are *always* opened directly right to the current one, instead of opening after the last tab opened tab from the current tab (many "tabs" in that sentence).

You could navigate to `about:profiles` to quickly create a new profile, install MAC and see if you can reproduce.

**Comment by RealDavidoff** [comment]
Okay thx, THAT is the browser behavior I'd love to have too (I guess most people would).
I may have memory loss but I believe *before* multi account container plugin (or discard?? plugin) it WAS like that, yes.
Thx for confirming it IS possible to achieve that.
